pallets/multisig/src/lib.rs

```diff
@@ -69,9 +69,9 @@
 //! an event.
 //! - `create_proposal` - Creates a proposal for a multisig transaction.
 //! - `create_or_approve_proposal` - Creates or approves a multisig proposal.
-//! - `approve_for` - Approves a multisig proposal and executes it if enough signatures have been
+//! - `unsafe_approve` - Approves a multisig proposal and executes it if enough signatures have been
```

Comment:
It is now made private, so you can remove its description from here.

Reply:
Yeah, I forgot to update the top documentation. I'll remove this and add the new functions.